plugin-server/tests/postgres/queue.test.ts

```diff
@@ -25,6 +25,7 @@ async function createTestHub(): Promise<[Hub, () => Promise<void>]> {
         PLUGINS_CELERY_QUEUE: 'ttt-test-plugins-celery-queue',
         CELERY_DEFAULT_QUEUE: 'ttt-test-celery-default-queue',
         LOG_LEVEL: LogLevel.Log,
+        KAFKA_ENABLED: false,
```

Comment:
I updated this test in my redisqueue removal pr, so whatever makes these tests pass here is fine and will be ignored 😅